### DISCUSSION TITLE
[GetConnect] Please add maximum retries

### DISCUSSION THREAD

**Discussion by nandhakumargdr** [jonataslaw/getx#849] [Ideas]
Hi,

It will be very useful if you add maximum retries for an HTTP request, so that when the request timed out or it fails because of a slow internet connection the request can be retried for a specified time.

`httpClient.maxRetries = 3`

Similar to 

`httpClient.maxAuthRetries = 3`

**Comment by saidino84** [comment]
Yeah

**Comment by mozaa-solana** [comment]
I don't know how to retry... 
Could you help me?

**Comment by jeidison** [comment]
Yeah
